src/main/java/Persistence/SaveFile.java

```diff
@@ -30,6 +30,16 @@ public boolean checkFile() {
         return this.file.isFile();
     }
 
+
+
+    public void saveGameNewFile(String[][] playerData, int[] mainStates) throws IOException{
```

Comment:
I think for now it should be fine to have each save create a new file. We could expand the feature set to include overwriting in the future but as an MVP I think what we have now is fine for functionality.

Reply:
I'm really just worried about how the GUI will work, that is, if we have so many options, how can we present them to the user? I'm not sure if we can get dropdown menus working, but if we could then it would be fine as it is.

Reply:
We can confirm with Harsh, it makes sense that we do not want to display an excessive amount of save files but I think the problem could be fixed more simply if we got a dropdown menu to work rather than refactor hundreds of lines of code at the current stage of our project.